Bloxstrap/JsonManager.cs

```diff
@@ -47,6 +47,9 @@ public virtual void Load(bool alertFailure = true)
 
                     if (!String.IsNullOrEmpty(message))
                         Frontend.ShowMessageBox($"{message}\n\n{ex.Message}", System.Windows.MessageBoxImage.Warning);
+                    
+                    // Create a backup of loaded file
+                    File.Copy(FileLocation, FileLocation + ".bak");
```

Comment:
This will fail if a backup file already exists.

Reply:
As I wrote in pr, what should the behaviour be when it fails?